tests/upgrade_path/test_upgrade_path.py

```diff
@@ -86,6 +86,12 @@ def setup_upgrade_test(duthost, localhost, from_image, to_image, tbinfo,
     # Install target image
     logger.info("Upgrading to {}".format(to_image))
     target_version = install_sonic(duthost, to_image, tbinfo)
+    # Remove config_db.json before cold-reboot to target image
+    # to ensure the new config_db.json is generated from minigraph
+    if "cold" in reboot_type \
+            and duthost.command("ls /host/old_config/minigraph.xml", module_ignore_errors=True)['rc'] == 0:
+        logger.info("Remove config_db.json before cold reboot to target image")
+        duthost.command("rm -f /host/old_config/config_db.json", module_ignore_errors=True)
```

Comment:
`install_sonic` does it already - https://github.com/bingwang-ms/sonic-mgmt/blob/a25951a3de94fc74dd9dba06d7d0669f90f0a084/ansible/library/reduce_and_add_sonic_images.py#L82

Reply:
Do we call `reduce_and_add_sonic_images.py` in the upgrade_path test?

Reply:
> Do we call `reduce_and_add_sonic_images.py` in the upgrade_path test?

Oh, yes, it's called. Then it's more interesting. I do see the `config_db.json` exist in `/host/old_config` after upgrading.